junction/profiles/admin.py

```diff
@@ -1,3 +1,10 @@
-# from django.contrib import admin
+from django.contrib import admin
 
 # Register your models here.
+from .models import Profile
+
+
+class UserAdmin(admin.ModelAdmin):
+    list_display = ('__unicode__', 'city', 'contact_no')
```

Comment:
This is to display user name in Profile admin. corresponding to whom the city and contact_no belongs.